src/Instance/MoveItemsContext/MoveItemsContext.js

```diff
@@ -50,6 +45,21 @@ const MoveItemsContext = ({ children, moveItems }) => {
     setActiveDropZone(undefined);
   }, [selectedItemsMap]);
 
+  const onSelect = useCallback(({ target }) => {
+    const from = target.dataset.itemId;
+    const to = target.dataset.toId;
+
+    itemsMovement(from, to, target);
```

Comment:
`moveItems`?

Reply:
moveItems is a function from props

Reply:
use another name, `itemsMovement` doesn't sound like a function